vendor.mod

```diff
@@ -145,18 +153,18 @@ require (
 	github.com/containerd/console v1.0.4 // indirect
 	github.com/containerd/go-cni v1.1.10 // indirect
 	github.com/containerd/go-runc v1.1.0 // indirect
-	github.com/containerd/nydus-snapshotter v0.14.0 // indirect
-	github.com/containerd/stargz-snapshotter/estargz v0.15.1 // indirect
-	github.com/containerd/ttrpc v1.2.5 // indirect
-	github.com/containernetworking/cni v1.2.2 // indirect
+	github.com/containerd/nydus-snapshotter v0.14.1-0.20241016012921-e55ae70fd45a // indirect
+	github.com/containerd/stargz-snapshotter/estargz v0.16.2 // indirect
+	github.com/containerd/ttrpc v1.2.6 // indirect
+	github.com/containernetworking/cni v1.2.3 // indirect
 	github.com/containernetworking/plugins v1.5.1 // indirect
 	github.com/cyphar/filepath-securejoin v0.3.5 // indirect
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
```

Comment:
sigh; do we know who decided to bump to master? There's literally _NO_ changes; https://github.com/davecgh/go-spew/compare/v1.1.1...d8f796af33cc

Reply:
I suspect this is Nydus as well;

```bash
go mod graph | grep ' github.com/davecgh/go-spew@v1.1.2'
github.com/docker/docker github.com/davecgh/go-spew@v1.1.2-0.20180830191138-d8f796af33cc
github.com/containerd/containerd/v2@v2.0.0 github.com/davecgh/go-spew@v1.1.2-0.20180830191138-d8f796af33cc
github.com/containerd/nydus-snapshotter@v0.14.1-0.20241016012921-e55ae70fd45a github.com/davecgh/go-spew@v1.1.2-0.20180830191138-d8f796af33cc
github.com/moby/buildkit@v0.18.2 github.com/davecgh/go-spew@v1.1.2-0.20180830191138-d8f796af33cc
```